docs/core/tutorials/testing-library-with-visual-studio.md

```diff
@@ -157,7 +157,7 @@ Your test run had no failures, but change it slightly so that one of the test me
 
    ![Test Explorer window showing the Is False assertion failure](./media/testing-library-with-visual-studio/failed-test-detail.png)
 
-1. Remove the code that you added (`"Error", `) and rerun the test. The tests will pass.
```

Comment:
Potentially this could have been suppress inline since the trailing space might be correct for the example. Losing the space shouldn't lessen understanding of the block to be removed from the sample

Reply:
Wondering if this could be rewritten a bit. Perhaps something like Undo the modification you did in step 1 and remove the string "Error". Would that make it more clear?

Reply:
I think both work. Third alternate: 'Remove the value "Error" from the array'
No real preference though, as I think they're both understanable

Reply:
I think this last suggestion or mine make the text more clear. Would you like to change that? Otherwise, I can merge the changes as-is as everything else looks good.

Reply:
Added your suggest text